lib/modules/apostrophe-pieces/lib/api.js

```diff
@@ -628,6 +628,8 @@ module.exports = function(self, options) {
         }
       });
     }
+    options.removeColumns = (options.defaultRemoveColumns || [ 'published' ])
```

Comment:
This is in apostrophe-pieces so you are removing published for everything. But I don't think you need it, since the subclass will have set this correctly already in beforeConstruct.